src/components/breedingModal.html

```diff
@@ -356,7 +356,9 @@ <h4><u>Hatchery</u></h4>
 
                     <p class="text-warning"><i>NOTE: Base attack is the value your Pokémon originally has at level 100, before any breeding buffs.</i></p>
 
-                    <p>Any Proteins applied to Pokémon will also give them an additional 1 attack per hatch.</p>
+                    <p>Any Proteins applied to Pokémon will give them an additional 1 attack per hatch, but require 20 more steps.</p>
```

Comment:
The 1 attack per hatch is no longer correct

Reply:
Sorry, I reverted it back to 1 attack per hatch, but now adds steps.
Refer dev chat for more info.

Reply:
I think you forgot to revert it. At least it looks like that in Egg.ts

Reply:
Was reverted in the other calculation, but not egg, fixed 👍